source/cloud-security/amazon/services/supported-services/security-hub.rst

```diff
@@ -0,0 +1,180 @@
+#. Set up the Wazuh integration for Amazon Security Hub.
+
+AWS configuration
+-----------------
+
+Enabling Security Hub
+^^^^^^^^^^^^^^^^^^^^^
+
+AWS Security Hub uses service-linked AWS Config rules to perform security checks for most controls. We advise to `configure AWS Config <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-setup-prereqs.html#securityhub-prereq-config>`_ as a prerequisite.
+
+You have two alternative ways to enable AWS Security Hub:
+
+-  `AWS Organizations integration <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-settingup.html#securityhub-orgs-setup-overview>`_: Recommended for multi-account and multi-region environments.
+-  `Manually <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-settingup.html#securityhub-manual-setup-overview>`__: Recommended for standalone accounts and when the integration with AWS Organizations is unnecessary.
+
+   .. thumbnail:: /images/aws/security-hub-set-up.png
+      :align: center
+      :width: 70%
+
+
+You must attach the following AWS managed policies to the IAM identity.
+
+-  `AWSSecurityHubFullAccess <https://docs.aws.amazon.com/securityhub/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-awssecurityhubfullaccess>`__ to access the Security Hub console and API operations.
+-  `AWSSecurityHubOrganizationsAccess <https://docs.aws.amazon.com/securityhub/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-awssecurityhuborganizationsaccess>`__ to enable and manage the Security Hub through the Organizations integration.
+
+.. thumbnail:: /images/aws/security-hub-policies.png
+   :align: center
+   :width: 70%
+
+We recommend using `central configuration <https://docs.aws.amazon.com/securityhub/latest/userguide/central-configuration-intro.html>`__ to set up and manage Security Hub for the organization. Central configuration lets the administrator customize security coverage for the organization.
+
+Types of Security Hub integration with EventBridge
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Integrating Security Hub with EventBridge allows storing Security Hub events and insights in S3 buckets. The AWS documentation describes the necessary steps on how to create an event rule in EventBridge both for `automatically sent findings <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-all-findings.html#securityhub-cwe-all-findings-predefined-pattern>`__ and for `custom actions to send findings and insight results <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-custom-actions.html#securityhub-cwe-define-rule>`__. EventBridge needs a target triggered when an event is received that matches the event pattern defined in the rule. Therefore, as a required previous step, it is necessary to create a Firehose stream in Amazon Data Firehose. For example, this is the case for the :ref:`Amazon WAF integration <amazon_waf>`.
```

Comment:
Integrating Security Hub with EventBridge allows storing Security Hub findings and insights in S3 buckets.

There are three types of events available as follows. Each type of event uses a specific `EventBridge event format <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-event-formats.html>`__. The Wazuh integration takes from the events every relevant ``detail`` field and value along with the ``detail-type`` value.

-  **Security Hub Findings - Imported**: Security Hub automatically sends events of this type to EventBridge. It includes all new findings as well as updates to existing findings. Each event contains a single finding.
-  **Security Hub Findings - Custom Action**: Security Hub sends events of this type to EventBridge when custom actions are triggered. The events are associated with the findings of the custom actions.
-  **Security Hub Insight Results**: This type of event is used to process the Security Hub Insights. You can use custom actions to send sets of insight results to EventBridge. Insight results are the resources that match an insight.

To send the last two types of events to EventBridge, you need to create a `custom action in Security Hub <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-custom-actions.html>`__. Please refer to the Security Hub documentation  to achieve this.

Find more information about each type of events in `Types of Security Hub integration with EventBridge <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-integration-types.html>`__.

To integrate Security Hub with EventBridge, you need to create a Firehose stream in Amazon Data Firehose. EventBridge needs a target. It triggers the target when it receives an event matching the event pattern defined in a rule. Check the :ref:`Amazon WAF integration <amazon_waf>` for a configuration example.

In addition, you need to create an event rule in EventBridge. The AWS documentation provides steps on how to configure this.

-  `Creating an event rule for automatically sent findings <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-all-findings.html#securityhub-cwe-all-findings-predefined-pattern>`__ 
-  `Defining a rule for using custom actions to send findings and insight results <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-custom-actions.html#securityhub-cwe-define-rule>`__